pysages/methods/umbrella_integration.py

```diff
@@ -0,0 +1,89 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2020-2021: PySAGES contributors
+# See LICENSE.md and CONTRIBUTORS.md at https://github.com/SSAGESLabs/PySAGES
+
+import jax.numpy as np
+from typing import Callable
+from pysages.backends import ContextWrapper
+from .harmonic_bias import HarmonicBias
+from .utils import HistogramLogger
+
+
+class UmbrellaIntegration(HarmonicBias):
+    def __init__(self, cvs, *args, **kwargs):
+        dummy = np.zeros(len(cvs))
+        super().__init__(cvs, dummy, dummy, args, kwargs)
```

Comment:
```suggestion
        kspring = center = np.zeros(len(cvs))
        super().__init__(cvs, kspring, center, args, kwargs)
```
Just to provide context for whoever is reading this.